app/livechat/imports/server/rest/inquiries.js

```diff
@@ -0,0 +1,49 @@
+import { Meteor } from 'meteor/meteor';
+
+import { API } from '../../../../api';
+import { hasPermission } from '../../../../authorization';
+import { Users } from '../../../../models';
+import { LivechatInquiry } from '../../../lib/LivechatInquiry';
+
+API.v1.addRoute('livechat/inquiries', { authRequired: true }, {
+	get() {
+		if (!hasPermission(this.userId, 'view-livechat-manager')) {
+			return API.v1.unauthorized();
+		}
+		const { offset, count } = this.getPaginationItems();
+		const { sort } = this.parseJsonQuery();
+		const cursor = LivechatInquiry.find({ status: 'open' }, {
```

Comment:
I know we decided to return the entire inquiry object, but due to the large number of agents available, it would be best to return only a few fields, such as:

```
   {
            "_id": "jz9WE69S9zmvpW2Fd",
            "rid": "Q3ZLHdXYmPk8PxfPm",
            "name": "Renato Becker",
            "ts": "2019-06-11T02:15:54.761Z",
            "status": "open"
    }
```